python/mxnet/ndarray_doc.py

```diff
@@ -37,6 +37,17 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class elemwise_addDoc(NDArrayDoc):
```

Comment:
Would it make sense to add this example in the C++ documentation without the python-ness?. 
I feel this is a simple example and does not have be repeated in every language binding.